modules/apps/frontend-theme/frontend-theme-classic/src/css/_custom.scss

```diff
@@ -1,6 +1,5 @@
 // @import 'components/navbars'; // To Verify
 // @import 'components/sidebar'; // To Verify
-// @import 'components/table'; // To Verify
```

Comment:
Not relevant change

![table-2](https://user-images.githubusercontent.com/46778114/107658521-7c31fb80-6c86-11eb-9813-fa6033e57f02.jpg)
![table-1](https://user-images.githubusercontent.com/46778114/107658509-789e7480-6c86-11eb-83e7-886816ea147e.jpg)